modules/n1ql/pages/n1ql-language-reference/advise.adoc

```diff
@@ -483,7 +487,7 @@ ADVISE SELECT airportname FROM `travel-sample` WHERE type = 'airport' AND geo.al
 ====
 [source,n1ql]
 ----
-ADVISE SELECT * FROM `travel-sample` WHERE type LIKE 'air%'or type LIKE 'rou%';
+ADVISE SELECT * FROM `travel-sample`.inventory.landmark WHERE city LIKE "Par%" OR city LIKE "Lon%";
```

Comment:
This is an example where the type field -> scope translation loses some expressiveness, isn't it?